include/flatbuffers/reflection_generated.h

```diff
@@ -53,7 +53,11 @@ enum BaseType {
   Vector = 14,
   Obj = 15,
   Union = 16,
-  Array = 17
+  Array = 17,
```

Comment:
As the name implies, this file is automatically generated, so you can't make changes here. You could add a `MaxBaseType` enum to the corresponding `.fbs` file though.

Reply:
Right. :) 

Do you really want that value to be called MaxBaseType rather than CountBaseTypes? I feel that Max is misleading because that value will not be a hard limit and it's not a valid type value in itself either.

Reply:
Won't this be a problem for evolution if we ever add a new type here? Since CountBaseType == 18 currently, but in the future it might be 19 or something else. I'm not sure if it matters in this case, but i just want to bring it up if it does.

Reply:
The confusing thing is that this generated file is checked in. And I'm not sure how to rebuild it - if i delete it, make fails to rebuild it. What are the steps for rebuilding it? I expect I have to rebuild it after changing the fbs file so I can checkin the regenerated file, right?

Reply:
I think I got it figured out. I need a generated file to build a new generated file. I thought the build would generate a file from scratch, but it expects a file already.

Reply:
Actually we already generate min/max enum values, but only if using `--scoped-enums` or not `--no-prefix`, and we are using the latter in `reflection --no-prefix` :( Looks like that would be too intrusive to change that now.

A manual max value is not great, I agree, but should not cause trouble?
`MaxBaseType` just sounded more in line with what the rest of the code base uses.

Reply:
I'll change the name then.

Reply:
I changed the name, but I also realized that I was wrong - the file does not get auto-generated by the build. I had to update it manually to enable the build, but I'd need to know the correct procedure for regenerating it. Can you tell me that?

Reply:
`reflection/generate_code.sh`

Reply:
Done.